docs/madr/2-sqlite.md

```diff
@@ -0,0 +1,89 @@
+# Using aos-sqlite Module
```

Comment:
If I were scanning the ADRs by title, this one might be a bit too broad. Consider:
> # Using aos-sqlite to manage data registered in the ANT Registry Process